netlogo-gui/src/main/app/InterfacePanel.scala

```diff
@@ -344,7 +329,16 @@ class InterfacePanel(val viewWidget: ViewWidgetInterface, workspace: GUIWorkspac
     }
   }
 
-  def keyPressed(evt: KeyEvent): Unit = { }
+  def keyPressed(evt: KeyEvent): Unit = {
```

Comment:
@Akarshit , is that possible to do here? I agree that would be a good change to make, but I know that it has been troublesome to handle conflicting undo/redo code for the command center in addition to the new undo/redo  functionality introduced in this PR.